src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/preprocess/QueryParser.java

```diff
@@ -22,15 +22,25 @@
 public class QueryParser {
   private static final Logger logger = LogManager.getLogger(QueryParser.class);
 
+  private static final String BODY_FIELD_REQUIRED_ERROR_MESSAGE =
+      "Property [" + BODY_FIELD + "] must be specified";
   private static final String FILED_MISMATCH_ERROR_MESSAGE =
       "Mismatch: Field configured in %s property [%s] is not present in query fields [%s]. Will not apply Kendra Intelligent Ranking.";
+  private static final String QUERY_PARSER_RESULT_LOG =
+      "Kendra Intelligent Ranker query parser result for query type [%s]: queryText: %s, bodyField: %s, titleField: %s";
+  private static final String QUERY_PARSER_RESULT_LOG_WITHOUT_TITLE =
+      "Kendra Intelligent Ranker query parser result for query type [%s]: queryText: %s, bodyField: %s";
```

Comment:
Do we want to enable logging for all queries sent to Kendra?

My big concern with that is a) the increase in the volume of stuff that we're logging and b) the risk of logging user queries without conscious action on the part of the admin.

Reply:
Good point, especially with (b)

How about making this a debug log and removing the query text, but keeping body and title field names? It's difficult to debug which fields got sent to Kendra, especially since the title is optional.

Reply:
I think I'm okay with keeping it at INFO level, as long as we remove the query text.

Reply:
Addressed